style.css

```diff
@@ -0,0 +1,627 @@
+  height: 170px;
+  margin: 10px 15px;
+  padding: 10px;
+  box-shadow: inset 0 0 0 1000px rgba(245, 18, 0, 0.6);
+}
+
+.about-bg2 {
+  background-image: url(images/picture2.jpg);
+  background-size: cover;
+  font-size: 20px;
+  font-weight: bold;
+  line-height: 1.8;
+  color: white;
+  opacity: 0.75;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+  height: 170px;
+  margin: 10px 15px;
+  padding: 10px;
+  box-shadow: inset 0 0 0 1000px rgba(245, 18, 0, 0.6);
+}
+
+footer {
+  background-color: #414246;
+}
+
+footer h3 {
+  text-align: center;
+  color: white;
+  width: 100%;
+  padding: 20px;
+}
+
+.partners {
+  display: flex;
+  flex-flow: wrap row;
+  justify-content: center;
+  align-items: center;
+  padding: 30px 20px;
+}
+
+.partners img {
+  width: 100px;
+  padding: 10px;
+  filter: invert(36%) sepia(101%) saturate(0%) brightness(102%) contrast(105%);
+}
+
+/* copy rights */
+
+.copy-write {
+  display: flex;
+  margin: 20px auto;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.copy-text {
+  font-size: 13px;
+  font-weight: bold;
+  color: var(--color-4);
+  padding: 10px 20px;
+  text-align: center;
+  line-height: 1.8;
+}
+
+.copy-img {
+  width: 33.33%;
+  height: 100%;
+  padding: 30px;
+}
+
+/* pop up */
+
+.mobileMenuToggle-container {
+  transform: scale(1);
+  height: 100%;
+  position: fixed;
+  z-index: 5;
+}
+
+#mobileMenuToggle {
+  font-size: 30px;
+  font-weight: 800;
+  color: #535353;
+  cursor: pointer;
+}
+
+#menu-overlay {
+  background-color: rgba(0, 0, 0, 0.7);
+  height: 100vh;
+  width: 0;
+  position: fixed;
+  z-index: 6;
+  top: 0;
+  left: 0;
+  backdrop-filter: blur(6px);
+  overflow-x: hidden;
+}
+
+.mobile-menu {
+  padding: 50px 30px;
+}
+
+.mobile-menu li {
+  list-style: none;
+  padding-bottom: 60px;
+}
+
+.mobile-menu li a {
+  text-decoration: none;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 34px;
+  color: rgba(255, 255, 255);
+}
+
+#mobileMenuClose {
+  position: fixed;
+  top: 50px;
+  right: 50px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: var(--color-2);
+  font-weight: 900;
+  border: none;
+  width: auto;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+/* pseudo class */
+
+.program-list li:hover {
+  border: 2px solid var(--color-3);
+  transform: scale(1.05);
+}
+
+.desktop-nav-list2 li:last-child {
+  border: var(--color-2) 4px solid;
+  padding: 10px;
+  align-self: center;
+}
+
+.desktop-nav-list2 li:hover {
+  transform: scale(1.2);
+}
+
+@media only screen and (max-width: 768px) {
+  .about-start,
+  .home {
+    padding-top: 60px;
+  }
+
+  .mobileMenuToggle-container {
+    margin: 20px;
+  }
+
+  .hide-mobile {
+    transform: scale(0);
+    height: 0;
+  }
+
+  .program-list li {
+    padding-left: 5px;
+  }
+
+  .program-list li i {
+    width: 20%;
+  }
+
+  .program-list li h4 {
+    width: 24%;
+  }
+
+  .program-list li p {
+    width: 48%;
+  }
+}
```

Comment:
**Suggested Changes**
- [ ] We should use a mobile-first approach. This is a desktop-first media query. Let us effect this change and follow a mobile first approach. You may research on the benefits for mobile first.